frontend/src/constants/media.ts

```diff
@@ -68,4 +68,4 @@ export const contentStatus = Object.freeze({
 } as const)
 
 export const searchPath = (type: SearchType) =>
-  `/search/${type === ALL_MEDIA ? "" : type}`
+  `/search${type === ALL_MEDIA ? "" : "/" + type}`
```

Comment:
So does nuxt 3 not have trailing slashes now? We went around and around trying to always use or not use trailing slashes. If we're changing it again, that should definitely be made clear.

Reply:
Thank you for flagging this, I'll re-add the trailing slash using `site.trailingSlash` setting in `nuxt.config`: https://nuxtseo.com/nuxt-seo/guides/trailing-slashes#enabling-trailing-slashes for consistency

Reply:
I was wrong about this.

Our current setup is inconsistent: we only add the trailing slash to the "All media" search endpoint (so, the URL is `https://openverse.org/search/?q=cat` with a slash between the path and the query), and all other endpoints do not have a trailing slash (so, image search URL is `https://openverse.org/search/image?q=cat, with no slash between the path and the query).

In this branch, we remove trailing slashes from the All media search URL, so now we have no trailing slashes in the frontend.

I think it should be okay to change the URL structure for the search route because we disable robots on it and make it not searchable by search engines.